src/main/java/thePackmaster/cards/dimensiongatepack/Knighthood.java

```diff
@@ -21,6 +21,7 @@ public Knighthood() {
         setFrame("knighthoodframe.png");
         exhaust = true;
         tags.add(CardTags.HEALING);
+        isEthereal = true;
```

Comment:
Yay! Makes it a bit harder to use, can't stall with it unless you have discard or exhume effects. That's a big step towards making it a more reasonable card, and we'll see how it plays.